pkg/cli/whoami/whoami.go

```diff
@@ -9,7 +9,8 @@ import (
 	v1 "k8s.io/api/authentication/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
```

Comment:
This is deprecated. `genericiooptions` needs to be used.

Reply:
But isn't `genericclioptions` used for PrintFlags? I see I can use `genericiooptions` for IOStream. Both are used right? Because `rollback` and `startBuild` also use this approach

Reply:
`genericclioptions` has been deprecated in favor of `genericiooptions`. So theoretically you can use `genericclioptions`. However, it is always preferable to use the recommended one (i.e. `genericiooptions`), since the other one will be removed entirely at some point.